### ISSUE TITLE
User must be able to walk away before giving a final answer

### ISSUE THREAD

**Issue by ua2004** [ua2004/Millionaire-3D#32]
During the question player can take the money he already won and walk away but only if he has not given the final answer yet. When he takes the money his total prize is displayed. This is the money he already won during the previous questions.

![image](https://cloud.githubusercontent.com/assets/8424744/20179866/ba9c9d70-a760-11e6-95cb-beed83903452.png)

**Comment by forestt** [comment]
On issue #6 user was able to stand up and walk around during the game and give an ansver while walking around. Now as I understand when user stands up his total prize should be displayed, so that means that game is over and when user again will sit the chair the new game will be started?

**Comment by ua2004** [comment]
No, this is a completely different thing. There should be a new GUI button "Walk Away" ("Забрати гроші") on money tree. It should be visible only when all lifelines are used. I will try to make graphic asset for that button later. When user clicks that button the game finishes and total prize money is displayed.

Of course, the "Walk Away" button is clickable only before user has given his final answer.
